VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -356,6 +356,13 @@ public void addAll(List<DiscussionComment> items) {
         notifyDataSetChanged();
     }
 
+    public void addToTop(List<DiscussionComment> items) {
```

Comment:
I think we should add an `onPartialPageLoaded()` method to the `InfiniteScrollUtils.PageLoadCallback` interface that just adds the list without hiding the progress bar. Since the endorsed comments are only added to an empty adapter, there is no need to ensure that it's added at the top.

---

`InfiniteScrollUtils` seems to depend on a scroll listener to append the progress bar row, instead of retaining it until the list is loaded, and using `getView()` calls on it to trigger the load as well like the `EndlessAdapter` library. This implementation doesn't really make sense, but it does the loader to retain control I guess.